src/build/__init__.py

```diff
@@ -64,7 +64,7 @@ def check_dependency(req_string, ancestral_req_strings=(), parent_extras=frozens
     if sys.version_info >= (3, 8):
         from importlib import metadata as importlib_metadata
     else:
-        import importlib_metadata
+        import importlib_metadata  # type: ignore
```

Comment:
Please add a quick comment here saying that we need the `# type: ignore` and linking to the mypy issue.

Otherwise, LGTM. Thanks :blush:

Reply:
Probably best to put the comment on the line above.

Reply:
Please see https://github.com/pypa/build/pull/213 instead. Thanks!